RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -799,14 +818,24 @@ boolean L12_filthworms()
 		use(1, $item[Filthworm Royal Guard Scent Gland]);
 	}
 
+	//if we can kill the queen we don't care about gland drops anymore. kill her and finish this
 	if(have_effect($effect[Filthworm Guard Stench]) > 0)
 	{
-		handleFamiliar("meat");
-		autoAdv(1, $location[The Filthworm Queen\'s Chamber]);
-		return true;
+		return autoAdv(1, $location[The Filthworm Queen\'s Chamber]);
 	}
 
-	if(!have_skill($skill[Lash of the Cobra]))
+	//if we can guarentee stealing the stench gland then no point in buffing item drop
+	if(auto_have_skill($skill[Lash of the Cobra]) && get_property("_edLashCount").to_int() < 31)
```

Comment:
check should be `< 30`. Unlikely to cause issues as there's really not that much we spam lash on but you never know.

Reply:
change made